### DISCUSSION TITLE
How to change Team Name?

### DISCUSSION THREAD

**Discussion by jmacd867** [akiver/cs-demo-manager#894] [Q&A] [answered]
Heyo, I was wondering if there was a way to change a team name of a match, i have 2 matches with the exact same roster of players but because in the matches they have two different names (team_player1 vs. ActualTeamName) the teams tab doesnt work for it when compiling map behaviors. Any help?

**Comment by akiver** [comment, marked as answer]
Right-click on the match and click on "Update team names"
![f](https://github.com/user-attachments/assets/668d0d6b-f410-4b95-9418-c86b821576ca)
